cmd/block-generator/generator/generate.go

```diff
@@ -154,9 +194,35 @@ func (g *generator) WriteGenesis(output io.Writer) {
 	output.Write(json.Encode(gen))
 }
 
-func (g *generator) generateTransaction(round uint64, intra uint64) (types.SignedTxnWithAD, error) {
-	// TODO: Distribute transactions according to configuration
-	return g.generatePaymentTxn(round, intra)
+func getTransactionOptions() []interface{} {
```

Comment:
I tried this but weightedSelection complains that []string != []interface{} in that case. Types are still used when processing the selection.

I didn't want to use string so that I had the option of using enum types, maybe different types for the different transaction types.